index.html

```diff
@@ -7,320 +7,466 @@
+          <li>
+            <img class="java_img" src="images/html.png" alt="">
+            <h4 class="script_text">HTML</h5>
+          </li>
+        </ul>
+        <ul class="java_board">
+          <li>
+            <img class="java_img" src="images/css.png" alt="">
+            <h4 class="script_text">CSS</h5>
+          </li>
+        </ul>
+
+        <ul class="lang_container">
+          <li>
+            <h3 class="lang">Frameworks</h3>
+            <img class="union3" src="images/Union_arrow.png">
+          </li>
+        </ul>
+        <img class="saperator" src="images/Separator Bottom.png" alt="">
+        <ul class="lang_container">
+          <li>
+            <h3 class="lang">Skills</h3>
+            <img class="skills" src="images/Union_arrow.png">
+          </li>
+        </ul>
+        <img class="saperator" src="images/Separator Bottom.png" alt="">
+      </div>
     </div>
+  </section>
 
+  <section class="about-me">
+    <div class="about">
+      <div class="about1">
+        <h2>About Myself</h2>
+        <h3>Hello I’m a software developer! I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.</h3>
+        <p>LET’S CONNECT</p>
+        <ul class="social">
+          <li><a href=""><img src="PLAY.png" alt=""></a></li>
+          <li><a href=""><img src="Linkedin icon.png" alt=""></a></li>
+          <li><a href=""><img src="two.png" alt="logo"></a></li>
+          <li><a href=""><img src="Twitter.png" alt=""></a></li>
+          <li><a href=""><img src="M.png" alt="logo"></a></li>
+        </ul>
+        <button>Get My Resume</button>
+      </div>
 
-    <ul class="lang_container">
-      <li>
-        <h3 class="lang">Languages</h3>
-        <img class="union3" src="images/Union_drop.png">
-      </li>
-    </ul>
-    <ul class="java_board">
-      <li>
-        <img class="java_img" src="images/Ellipse 1 (1).png" alt="">
-        <h4 class="script_text">JavaSript</h5>
-      </li>
-    </ul>
-    <ul class="java_board">
-      <li>
-        <img class="java_img" src="images/html.png" alt="">
-        <h4 class="script_text">HTML</h5>
-      </li>
-    </ul>
-    <ul class="java_board">
-      <li>
-        <img class="java_img" src="images/css.png" alt="">
-        <h4 class="script_text">CSS</h5>
-      </li>
-    </ul>
-    <ul class="lang_container">
-      <li>
-        <h3 class="lang">Frameworks</h3>
-        <img class="union3" src="images/Union_arrow.png">
-      </li>
-    </ul>
-    <img class="saperator" src="images/Separator Bottom.png" alt="">
-    <div class="lang_container">
-      <ul class="lang_container">
-        <li>
-          <h3 class="lang">Skills</h3>
-          <img class="skills" src="images/Union_arrow.png">
-        </li>
-      </ul>
-      <img class="saperator" src="images/Separator Bottom.png" alt="">
+      <div class="about2">
```

Comment:
- [ ] For the about section Kindly make sure it is separated from the Work section by using the right background color (Work-section:#E5E5E5;,About:#fff)
- [ ] Kindly make sure to resize your About section's elements to match the Figma design as shown below :point_down: 

[Screencast from 04-03-2023 23:42:17.webm](https://user-images.githubusercontent.com/97834160/222929997-1641b77f-46fc-469f-8257-d616316970e8.webm)
![Screenshot from 2023-03-04 23-47-45](https://user-images.githubusercontent.com/97834160/222930079-335a33c9-8cda-43ea-b956-a517370d13b3.png)

Reply:
As the previous reviewer requested, please fix the background colors. Works section and About Myself section both have light color but slightly different.
<img width="958" alt="Screenshot 2023-03-05 at 1 45 22 AM" src="https://user-images.githubusercontent.com/25887253/222932137-ca655b65-9c7c-4b43-95f5-114a1e4f9b24.png">
<img width="1440" alt="Screenshot 2023-03-05 at 1 46 07 AM" src="https://user-images.githubusercontent.com/25887253/222932138-93927ee2-4c98-4da5-aea0-f2b8f73a1a8d.png">